charts/folding-at-home/Chart.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+name: folding-at-home
+description: A helm chart for deploying Folding @ Home workers
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application.
+appVersion: 1.16.0
+icon: file://icon.svg
```

Comment:
Make sure to update this if you decide to change the directory tree structure to include the v0.1.0 subfolder.

Reply:
good call